resources.js

```diff
@@ -153,7 +154,9 @@ function loadResource(name,max,rate,tradable,stackable,color) {
             rate: rate
         };
     }
-
+    else{
+        global['resource'][name].name = name === races[global.race.species].name? name : name === 'Money' ? '$' : loc('resource_' + name + '_name')
```

Comment:
same issue as above

Reply:
One of the resources have the name of the race: 
line 104: loadResource(races[global.race.species].name,0,0,false,false,'warning');
So i need check if it is that resource before call loc('resource_' + name + '_name')

Reply:
Ah ok